ntc_templates/templates/index

```diff
@@ -562,6 +562,7 @@ ubiquiti_edgerouter_show_ipv6_neighbors.textfsm, .*, ubiquiti_edgerouter, sh[[ow
 ubiquiti_edgerouter_show_dhcp_leases.textfsm, .*, ubiquiti_edgerouter, sh[[ow]] dhcp leases
 ubiquiti_edgerouter_show_interfaces.textfsm, .*, ubiquiti_edgerouter, show interfaces
 ubiquiti_edgerouter_show_ipv6_route.textfsm, .*, ubiquiti_edgerouter, show ipv6 route
+ubiquiti_edgerouter_show_nat_rules.textfsm, .*, ubiquiti_edgerouter, show nat rules
```

Comment:
We haven't done it for the other ones, but it might be better to use the : `sh[[ow]]` like upper.
It tells NTC to parse also when using sh, sho, show.